src/Runner.Worker/ActionManager.cs

```diff
@@ -795,97 +792,50 @@ private async Task DownloadRepositoryActionAsync(IExecutionContext executionCont
             string link = downloadInfo?.TarballUrl;
 #endif
 
-            Trace.Info($"Save archive '{link}' into {archiveFile}.");
```

Comment:
the diff is not readable.
I break the change into 3 commits, which might explain the diff better.
- https://github.com/actions/runner/pull/2857/commits/41a2d7cb15c3bffb83866cddf2039b04005ac103  move code around and should not change any behavior.
- https://github.com/actions/runner/pull/2857/commits/f687e155e5e1ec0ab0ddd7c66a9e52bdd99b145f change behind FF that use local actions cache instead of downloading from the service.
- https://github.com/actions/runner/pull/2857/commits/95ee6ccda1a08d43d79603f721ec2443d978c724 cleanup.